elements/bulbs-carousel/bulbs-carousel.js

```diff
@@ -0,0 +1,174 @@
+import invariant from 'invariant';
+import { BulbsHTMLElement, registerElement } from 'bulbs-elements/register';
+import { moveChildren } from 'bulbs-elements/util';
+
+import './bulbs-carousel.scss';
+
+export const supportsCalcInTransform = (() => {
+  let div = document.createElement('div');
+  div.style.transform = 'translateX(calc(100% - 10px))';
+  return div.style.transform !== '';
+})();
+
+export default class BulbsCarousel extends BulbsHTMLElement {
```

Comment:
The interface of this object seems to be unclearly defined.

Probably most of the time we'd look at the source to figure out what to access, but if we listed out the properties here, wouldn't it show, what are presumably, internal properties--e.g. `slider`, `currentIndex`, `slider`, `handleClick`--mixed in with interface properties--e.g. `carouselItems`, `updateCurrentIndex`, `slideToNext`?

Can some of these properties be hidden via closure or some other strategy? Maybe something to discuss in a larger meeting about how we want to use classes.